ipatests/test_integration/test_adtrust_install.py

```diff
@@ -199,3 +199,14 @@ def test_add_agent_on_running_replica_with_compat(self):
         entry = conn.get_entry(DN(
             "cn=groups,cn=Schema Compatibility,cn=plugins,cn=config"))
         assert entry.single_value['schema-compat-lookup-nsswitch'] == "group"
+
+        """Test if schema-compat-entry-attribute is set
+
```

Comment:
Same remark as the non-AD case. Please move this to a dedicated test.

Reply:
Also it'd be clearer if the test function names matched what the test did. This is not the case here with the additional test.

Reply:
@fcami Hello, 
I moved test as suggested for non AD installation however IMO, AD one should remain like this as we are interested in only one attribute that the `entry` variable providing already.

Reply:
Moved both as suggested.